universal-login-react/src/ui/TopUp/Fiat/FiatFooter.tsx

```diff
@@ -34,8 +34,14 @@ export const FiatFooter = ({paymentMethod, walletService, selectedCurrency}: Fia
             <InfoText>You can pay with any UK bank or Revolut</InfoText>
             <img src={RevolutLogo} srcSet={RevolutLogo2x} className="revolut-logo" alt="Revolut" />
           </div>
-          {minimumAmount && <div className="info-block info-row">
-            <InfoText>Minimum amount is {minimumAmount} {selectedCurrency}</InfoText>
+          {minimumAmounts && minimumAmounts.generalMinimalAmount === minimumAmounts.minimalAmountForRevolut && <div className="info-block info-row">
```

Comment:
It can be simpler:
```ts
{minimumAmounts && minimumAmounts.generalMinimalAmount === minimumAmounts.minimalAmountForRevolut ? 
    <div className="info-block info-row">
        <InfoText>
            Minimum amount is {minimumAmounts.generalMinimalAmount} {selectedCurrency}
        </InfoText>
    </div>
: <div className="info-block info-row">
        <InfoText>
            Minimum amount is {minimumAmounts.generalMinimalAmount} {selectedCurrency} or {minimumAmounts.generalMinimalAmount} {selectedCurrency} if you pay with Revolut <img src={RevolutLogo} srcSet={RevolutLogo2x} className="revolut-logo" alt="Revolut" />
        </InfoText>
    </div>}
```

Reply:
Yeah, done